### ISSUE TITLE
Ensure WBbt is coherent when combined with Uberon

### ISSUE THREAD

**Issue by cmungall** [obophenotype/uberon#1405]
Moved from https://github.com/geneontology/simple-annoton-editor/issues/32

DNC and VNC are unsat:

![image](https://user-images.githubusercontent.com/50745/37003638-9fba83e0-2082-11e8-8029-387e09c441c5.png)

this is probably because of an inappropriate taxon equivalence for somatic nervous system @raymond91125

**Comment by raymond91125** [comment]
I need to add the definition in. Somatic nervous system in the nematode actually means 'Part of the nervous system that is NOT pharyngeal'.

**Comment by cmungall** [comment]
ok, good. It's definitely different from vertebrate SNS

**Comment by balhoff** [comment]
`head muscle`:

![head_muscle](https://user-images.githubusercontent.com/210210/37040790-a4c15732-2128-11e8-8294-a90a89bdc232.png)

**Comment by balhoff** [comment]
`Z4.appaaaa nucleus` (and also `Z4.appaaap nucleus`):

![z4](https://user-images.githubusercontent.com/210210/37041059-4436c702-2129-11e8-889e-1b4450872cd5.png)

**Comment by cmungall** [comment]
I LOVE THAT WORM HAS A ZAPPA NUCLEUS

The spermathecum thing came up before: #367

we weakened the restriction that they were only in females, as they are found in worm hermaphrodites. We swapped for a never-in male reproductive system. This is still too strong I think? It should never be never-in male (as herms have both M and F reproductive systems).

I love using a reasoner with worms we find such interesting stuff! cc @vanaukenk @raymond91125 

On this  topic, I think we should follow @dosumis suggestion from way back and have more specific terms for insect and worm spermathecum, in uberon it's currently a loose grouping.

I note that the GO definition is fly-centric

```
def: "The process in which the anatomical structures of a spermathecum, a sperm storage organ, are generated and organized. Paired spermathecae lie at the anterior end of the insect uterus on the dorsal side. Each spermatheca consists of an oval shaped capsule, connected to the uterus by a spermathecal stalk." [PMID:12679097]
comment: See also the fly_anatomy.ontology term 'spermathecum ; FBbt:00004921'.
```

the core of the def is generic (if this is the def then I think the name 'sperm storage organ' is better). But the gloss of the def is fly centric. We need to be clearer with our defs.

Note it's been used for worm and flies:
http://amigo.geneontology.org/amigo/term/GO:0035211
